frontend/apps/web/app/globals.css

```
@@ -1,3 +1,27 @@
+@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.css');
+
+@font-face {
+  font-family: 'GmarketSans';
```

Stylelint (17.11.0)

[error] Expected no quotes around "GmarketSans" (font-family-name-quotes)

(font-family-name-quotes)